src/bttrx_wifi.cpp

```diff
@@ -104,7 +106,7 @@ void BTTRX_WIFI::handleSet(AsyncWebServerRequest *request)
   	value = request->getParam("value")->value().c_str();
 	}
 
-	if (name.empty() || value.empty()) {
+	if (name.empty()) {
```

Comment:
OK, but it breaks error handling for the other parameters a bit...so we have to enforce checking for unallowed (i.e. empty) values in each and every `handleSet...` method in bttrx_control. But that's fine for me.

Reply:
Another option is to define parameter to be like "none" or the like as a fixed value which is considered to delete the callsign then? This way the callsign can never be just empty.